app/views/management/schools/report.html.erb

```diff
@@ -14,7 +14,12 @@
 
 <% @overview_charts.each_with_index do |(energy, chart_config), index| %>
   <h2><%= t("management.schools.overview_charts.overview.#{energy.to_s}") %></h2>
-  <%= chart_tag(@school, chart_config[:chart], no_zoom: true, chart_config: {y_axis_units: select_y_axis(@school, chart_config[:chart], chart_config[:units])}) %>
+  <%= component 'chart',
```

Comment:
Looking at the existing page, this printable version of the dashboard doesn't have the axis controls or analysis controls, so I think these options should be passed into the component?